slick/src/main/scala/slick/jdbc/OracleProfile.scala

```diff
@@ -61,6 +67,8 @@ trait OracleProfile extends JdbcProfile {
 
   override protected def computeCapabilities: Set[Capability] = (super.computeCapabilities
     - RelationalCapabilities.foreignKeyActions
+    - RelationalCapabilities.createIfNotExists
+    - RelationalCapabilities.dropIfExists
```

Comment:
The `-` here means you are removing these capabilities from the set of supported capabilities for this driver, which seems to be inaccurate. (Same for the other drivers, where you implemented the feature).